### ISSUE TITLE
Home Button doesn't work, Touch ID works

### ISSUE THREAD

**Issue by Panak252** [leminlimez/Cowabunga#97]
After changing the default subtype and waiting a couple hours, my iPhone 7 Plus's home button stopped waking up the display, can't exit apps with it, and can't trigger screenshots or open the app switcher. iOS 15.6.1, never jailbroken this one. Home button was working before changing default SubType. Can't change it back either, mentioned on other issue.

**Comment by Panak252** [comment]
I mostly would just like the button to work in order to wake up the screen, as double tapping doesn't wake it up, and the home button is more convenient to wake the phone up, rather than pressing the power button or tilting it to trigger raise to wake. Just something that would be neat.

**Comment by leminlimez** [comment]
it is an issue with x gestures

i am working on a fix for the resetting the default subtype issue

**Comment by Panak252** [comment]
Thanks for the quick reply, and please take your time, no pressure :)

**Comment by Panak252** [comment]
I thought I'd post what I've found out from testing. Starting off, I've got the normal navigation, on the 7+ ( the home button).
If when I have the home button enabled, I go and change the subtype to the iPhone X Gestures, and then respring, then the home button will continue working, along with the iPhone X gestures, simultaneously.

If I have the iPhone X Gestures, and restart my device, then the home button is disabled. Touch ID still works, but pressing the home button does absolutely nothing. (And obviously I have the X Gestures)

So, in order to have a working home button, as well as having the iPhone X gestures, I have to have set my subtype to default, restart (or not, if the home button is working and without the gestures already) set my subtype to the iPhone X Gestures, and then respring. And tada, you now have yourself a home button iPhone, with both the gestures and the home button.

As far as why someone would want both working, I've got an answer for you.

Waking up the phone from sleep, it's easier to use the home button, than using the power button. It doesn't seem like much, and you might just say "press the power button, or use raise to wake", but after testing this for a while (months), I've found using raise to wake and / or using the power button to wake the phone from sleep, to be very annoying, especially if I have a case on (and it's not thick).

Hope this is of help @leminlimez :)
Best of luck
